hw/ip/pinmux/doc/theory_of_operation.md

```diff
@@ -41,6 +41,12 @@ However, only the wakeup detector logic will be actively clocked during sleep in
 
 See the [retention logic](#retention-logic) and [wakeup detectors](#wakeup-detectors) sections for more details about the mux implementation.
 
+### USB Wakeup Detection Module
```

Comment:
@alees24 @a-will I have added a reference to the USB device documentation here noting that the pinmux contains the USB wakeup module in the Earlgrey top-level configuration.